sway/desktop/fx_renderer/fx_framebuffer.c

```diff
@@ -5,7 +5,17 @@ void fx_framebuffer_bind(struct fx_framebuffer *buffer) {
 	glBindFramebuffer(GL_FRAMEBUFFER, buffer->fb);
 }
 
-void fx_framebuffer_create(struct fx_framebuffer *buffer, int width, int height, bool bind) {
+void fx_framebuffer_init(struct fx_framebuffer *buffer) {
+	buffer->fb = -1;
+	buffer->stencil_buffer = -1;
+	buffer->texture.id = 0;
+	buffer->texture.target = 0;
+	buffer->texture.width = -1;
+	buffer->texture.height = -1;
+}
+
+void fx_framebuffer_create(struct fx_framebuffer *buffer, int width, int height,
+		bool bind, bool create_stencil_buffer) {
```

Comment:
what are your thoughts on removing the bind argument? It'd clean up the creation function a bit, and make it a bit more clear elsewhere in the codebase when we are binding a buffer as we'd need to explicitly call that function after creation.

Reply:
I 100% agree, I'm not sure if I like the `create_stencil_buffer` either. Any ideas? Maybe a `fx_framebuffer_create_stencil` instead?